src/shared/views/Login.js

```diff
@@ -240,6 +256,7 @@ class _Login extends React.Component<Props, State> {
                     projectContributionCount: 0,
                     taskContributionCount: 0,
                     username,
+                    usernameKey: username,
```

Comment:
Let's add a NOTE why usernameKey = username in this case.

Reply:
The comment is not correct. Use this instead:
We can assign username to usernameKey because we have already validated the username earlier